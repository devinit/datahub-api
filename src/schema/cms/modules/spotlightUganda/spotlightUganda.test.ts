import {getSpotlightUgandaPageData} from '.';
import * as prettyFormat from 'pretty-format';

describe('Spotlight Uganda module tests', () => {
    it('should return page data for Spotlight Uganda', async () => {
        const pageData: DH.IPage[] = await getSpotlightUgandaPageData();
        expect(pageData.length).toBeGreaterThan(0);
        expect(prettyFormat(pageData)).toMatchSnapshot();
    }, );
});
