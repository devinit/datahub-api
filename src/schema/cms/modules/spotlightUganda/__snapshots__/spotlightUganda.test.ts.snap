// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spotlight Uganda module tests should return page data for Spotlight Uganda 1`] = `
"Array [
  Object {
    \\"id\\": \\"introduction\\",
    \\"meta\\": \\"\\",
    \\"narrative\\": \\"\\",
    \\"title\\": \\"Spotlight on Uganda is a comprehensive source of Uganda's financial resource flow data at the sub-national (district) level, alongside indicators on poverty, population, education, health, water, hygiene and sanitation. It highlights the geographical variance in sector performance and financial resources, and seeks to answer whether resources are allocated according to need. Explore the country picture by selecting topics and click on a district for an in-depth profile\\",
  },
]"
`;
